third_party/terraform/tests/resource_dataproc_cluster_test.go.erb

```diff
@@ -722,6 +722,26 @@ func TestAccDataprocCluster_withKerberos(t *testing.T) {
 	})
 }
 
+func TestAccDataprocCluster_withAutoscalingPolicy(t *testing.T) {
+	t.Parallel()
+
+	rnd := randString(t, 10)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDataprocClusterDestroy(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataprocCluster_withAutoscalingPolicy(rnd),
+			},
+			{
+				Config: testAccDataprocCluster_removeAutoscalingPolicy(rnd),
+			},
```

Comment:
I suppose we don't need a check here because we are just making sure a diff doesn't exist after applying?

Reply:
Ideally we would have an import step, but we don't actually support import yet for dataproc clusters. I suppose it probably makes sense to check the object though to make sure removing the autoscaling policy actually removes it (vs just removing it from state). Will do shortly.